packages/harmony/src/styles/theme.ts

```diff
@@ -0,0 +1,153 @@
+import type { Theme, ThemeColors } from './types'
+
+export const themeColorsMap: Record<Theme, ThemeColors> = {
+  day: {
+    static: {
+      white: '#FFFFFF',
+      primary: '#CC0FE0'
```

Comment:
is there any way we can reference the css variables or vice-versa?

Reply:
I don't think you can, but I can look into it and see if there is a clean way to do it. It would be helpful for sure